webapi/src/main/scala/org/knora/webapi/store/iiif/impl/SipiServiceLive.scala

```diff
@@ -73,12 +75,15 @@ final case class SipiServiceLive(
    * @return a [[FileMetadataSipiResponse]] containing the requested metadata.
    */
   override def getFileMetadata(filePath: String): Task[FileMetadataSipiResponse] =
```

Comment:
Would be nice to have named methods for`get from temp` and `get from permanent` and only provide the `internalFilename` as parameter.

Reply:
done